src/components/JobsPanelParameters/jobsPanelParametersReducer.js

```diff
@@ -26,12 +26,7 @@ export const jobsPanelParametersReducer = (state, { type, payload }) => {
     case parametersActions.REMOVE_NEW_PARAMETER_DATA:
       return {
         ...state,
-        newParameter: {
-          name: '',
-          value: '',
-          parameterType: panelData.newParameterType[0].id,
-          valueType: 'string'
-        }
+        newParameter: { ...initialState.newParameter }
```

Comment:
When I had to change the `'string'` value to `'str'` as the default parameter type I noticed that it is duplicated and I actually changed it only in one place, and after seeing in the UI that the old value is still displayed in some flow, I looked for it and found the second place it is declared.
This here is a fix so it's more maintainable, so if the initial value changes, we can change it in one place only.